src/tests/stub_carp.cc

```diff
@@ -16,4 +16,5 @@
 CachePeer *carpSelectParent(PeerSelector *) STUB_RETVAL(nullptr)
 void carpReset(Configuration::SmoothReconfiguration &) STUB
 void carpResetIfChanged(Configuration::SmoothReconfiguration &, const CachePeer &, const CachePeer &) STUB
+void carpInit() STUB
```

Comment:
This fixes the `testHttpRange` test. We need this and do not need similar stubs for sourcehash/userhash because the test includes `peer_sourcehash.cc` and `peer_userhash.cc`, but lacks `carp.cc`.

Reply:
I see. That inclusion pattern is inconsistent, but we are not going to fix that problem in this PR. Thank you for researching this.